src/main/java/embl/ebi/variation/eva/pipeline/jobs/InitDBConfiguration.java

```diff
@@ -32,17 +33,22 @@
 
 @Configuration
 @EnableBatchProcessing
-@Import({GenesLoad.class})
-public class InitDBConfiguration {
+@Import({IndexesCreate.class, GenesLoad.class})
+public class InitDBConfiguration extends CommonJobStepInitialization {
 
     private static final Logger logger = LoggerFactory.getLogger(InitDBConfiguration.class);
     public static final String jobName = "initialize-database";
+    public static final String GENERATE_DATABASE_INDICES = "Generate database indices";
```

Comment:
Replace all occurrences of "indices" with "indexes", for the sake of consistency.